zebra-consensus/src/chain/tests.rs

```diff
@@ -320,78 +311,31 @@ async fn verify_fail_add_block_checkpoint_test() -> Result<(), Report> {
 async fn verify_fail_add_block_checkpoint() -> Result<(), Report> {
     zebra_test::init();
 
-    let block =
-        Arc::<Block>::zcash_deserialize(&zebra_test::vectors::BLOCK_MAINNET_GENESIS_BYTES[..])?;
-    let hash: BlockHeaderHash = block.as_ref().into();
-
     let (mut chain_verifier, mut state_service) = verifiers_from_network(Mainnet);
 
-    /// SPANDOC: Make sure the verifier service is ready (1/2)
+    /// SPANDOC: Make sure the block verifier service is ready (1/2)
     let ready_verifier_service = chain_verifier.ready_and().await.map_err(|e| eyre!(e))?;
-    /// SPANDOC: Set up the future to verify the block for the first time
-    let verify_future = timeout(
-        Duration::from_secs(VERIFY_TIMEOUT_SECONDS),
-        ready_verifier_service.call(block.clone()),
-    );
-    /// SPANDOC: Verify the block for the first time
-    // TODO(teor || jlusby): check error kind
-    let verify_response = verify_future
-        .map_err(|e| eyre!(e))
-        .await
-        .expect("timeout should not happen")
-        .expect("block should verify");
 
-    assert_eq!(verify_response, hash);
+    let transcript = Transcript::from(BLOCK_VERIFY_TRANSCRIPT1.iter().cloned());
+    transcript.check(ready_verifier_service).await.unwrap();
 
-    /// SPANDOC: Make sure the state service is ready (1/2)
+    /// SPANDOC: Make sure the state verifier service is ready (1/2)
```

Comment:
Nit: The state service is not a verifier:

```suggestion
    /// SPANDOC: Make sure the state service is ready (1/2)
```